core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2866,8 +2919,7 @@ private PhysicalOperation createDynamicFilterSourceOperatorFactory(
                         Type type = buildSource.getTypes().get(index);
                         return new DynamicFilterSourceOperator.Channel(filterId, type, index);
                     })
-                    .collect(Collectors.toList());
-            boolean isReplicatedJoin = isBuildSideReplicated(node);
+                    .collect(toImmutableList());
             boolean isBuildSideSingle = context.getDriverInstanceCount().orElse(1) == 1;
```

Comment:
Right, I've updated this so that `isBuildSideSingle` is always `false` for tardigrade DF. This was introduced for increasing DF limits when there is a gathering local exchange on build side of join. Since our DF collection takes place before this, it should always be `false` for the new code path.